specs/address_book_list/01_main.rst

```diff
@@ -0,0 +1,140 @@
++--------+------------------+
+| issue  | title            |
++--------+------------------+
+| 81_    | Address Book List|
++--------+------------------+
+
+.. sectnum::
+
+.. _81: https://github.com/gnosis/safe/issues/81
+
+
+Address Book List
+=================
+
+.. contents:: Table of Contents
+    :local:
+    :depth: 2
+
+Problem Definition
+-------------------------------
+
+- User can recognize names better than full addresses
+- User wants to associate addresses to names
+- User can see the address book entries and their details
+- User can search the address book entries
+- The address book is not synced (ie.: it is part of the local storage only)
+
+Inputs
+-----------
+
+- Address book entries stored locally (local database)
+- Search with name or Ethereum address
+
+Outputs
+------------
+
+- Formatted list to be displayed to the user. Each entry has:
```

Comment:
How about ENS names. Does it make sense to also display them? 
How do we make sure to differentiate ENS names from address book names?

Reply:
I would not consider ENS in this spec.

As mentioned above, maybe we should have a separate ticket on displaying addresses and related information.